src/components/OrgSettings/AgendaItemCategories/OrganizationAgendaCategory.module.css

```diff
@@ -13,16 +13,32 @@
   gap: 10px;
 }
 
-.btnsContainer .btnsBlock {
-  display: flex;
-  gap: 10px;
+.btnsContainer .input {
+  /* Styling for the input field */
+  flex: 1;
+  min-width: 18rem;
+  position: relative;
 }
 
-.btnsContainer button {
-  display: flex;
-  align-items: center;
+.btnsContainer input {
+  /* Styling for the input border */
+  outline: 1px solid var(--bs-gray-400);
```

Comment:
Please migrate this stylesheet to:
- https://github.com/PalisadoesFoundation/talawa-admin/blob/develop-postgres/src/style/app.module.css

Use only pre-existing variables as much as possible